skill-package/interactionModels/custom/en-US.json

```diff
@@ -320,22 +320,12 @@
                     "values": [
                         {
                             "name": {
-                                "value": "no",
-                                "synonyms": [
-                                    "negative",
-                                    "false"
-                                ]
+                                "value": "no"
                             }
                         },
                         {
                             "name": {
-                                "value": "yes",
-                                "synonyms": [
-                                    "obviously",
-                                    "of course",
-                                    "true",
-                                    "yeah"
```

Comment:
I removed the synonyms for yes and no because Alexa can just prompt the user to say yes or no if they were to say a synonym and requires less code.